src/pages/Share/ShareDetailsPage.tsx

```diff
@@ -0,0 +1,194 @@
+                    Report.openReport(
+                        report.reportID,
+                        '',
+                        displayReport.participantsList?.filter((u) => u.accountID !== currentUserID).map((u) => u.login ?? '') ?? [],
+                        optimisticReport,
+                        undefined,
+                        undefined,
+                        undefined,
+                        undefined,
+                    );
+                }
+                if (report.reportID) {
+                    if (isTextShared) {
+                        Report.addComment(report.reportID, message);
+                    } else {
+                        Report.addAttachment(report.reportID, file, message);
+                    }
+                }
+
+                const routeToNavigate = ROUTES.REPORT_WITH_ID.getRoute(reportID);
+                Navigation.navigate(routeToNavigate, CONST.NAVIGATION.TYPE.UP);
+            },
+            () => {},
+        );
+    };
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom
+            shouldEnableKeyboardAvoidingView={false}
+            shouldEnableMinHeight={DeviceCapabilities.canUseTouchScreen()}
+            testID={ShareDetailsPage.displayName}
+        >
+            <View style={[styles.flex1, styles.flexColumn, styles.h100]}>
+                <HeaderWithBackButton
+                    title={translate('share.shareToExpensify')}
+                    shouldShowBackButton
+                />
+                {!!report && (
+                    <>
+                        <View style={[styles.optionsListSectionHeader, styles.justifyContentCenter]}>
+                            <Text style={[styles.ph5, styles.textLabelSupporting]}>{translate('common.to')}</Text>
+                        </View>
+                        <UserListItem
+                            item={displayReport}
+                            isFocused={false}
+                            showTooltip={false}
+                            onSelectRow={() => {}}
+                            pressableStyle={[styles.flexRow]}
+                            shouldSyncFocus={false}
+                            isDisabled
+                        />
+                    </>
+                )}
+
+                <View style={[styles.ph5, styles.flex1, styles.flexColumn]}>
+                    <View style={styles.pv5}>
+                        <ScrollView>
+                            <TextInput
+                                value={message}
+                                multiline
+                                onChangeText={setMessage}
+                                accessibilityLabel={translate('share.messageInputLabel')}
+                                label={translate('share.messageInputLabel')}
+                            />
+                        </ScrollView>
+                    </View>
+                    {!isTextShared && (
```

Comment:
should this be !isTextShared? 

Maybe we should use variables like `shouldShow<DescriptiveName>` for these conditional render blocks